ports/stm32/boards/NUCLEO_G474RE/mpconfigboard.h

```diff
@@ -5,7 +5,7 @@
 #define MICROPY_HW_ENABLE_RNG       (1)
 #define MICROPY_HW_ENABLE_ADC       (1)
 #define MICROPY_HW_ENABLE_DAC       (1) // A4, A5
-#define MICROPY_HW_ENABLE_USB       (0) // A12 (dp), A11 (dm)
+#define MICROPY_HW_ENABLE_USB       (1) // A12 (dp), A11 (dm)
```

Comment:
I suggest this follow the NUCLEO_G0 board which also doesn't have a USB connector:
```
#define MICROPY_HW_ENABLE_USB       (0) // can be enabled if USB cable connected to PA11/PA12
```

Reply:
Yes, definitely, we should leave it off by default, and that makes sense to update the comment.

Sorry I just left it enabled in this PR for @jgpeiro to test.